packages/helm/symphony/values.yaml

```diff
@@ -24,8 +24,9 @@ cert-manager:
 flavor: oss
 redis:
   enabled: true
+  asSidecar: false
   image: redis/redis-stack-server:latest
-  port: 6379
+  port: "6379"
```

Comment:
I saw the port will be used in redis deployment yaml, can this value be string?

Reply:
I have changed it back for consistency. And also add port to redis config to allow port change.